tests/operator_test.go

```diff
@@ -1373,6 +1347,112 @@ var _ = Describe("ALL Operator tests", func() {
 	})
 })
 
+func getCDICrAndPods(f *framework.Framework) (*cdiv1.CDI, *corev1.PodList) {
+	cdiPods, err := f.K8sClient.CoreV1().Pods(f.CdiInstallNs).List(context.TODO(), metav1.ListOptions{})
+
+	Expect(err).ToNot(HaveOccurred(), "failed listing cdi pods")
+	Expect(len(cdiPods.Items)).To(BeNumerically(">", 0), "no cdi pods found")
+
+	cr, err := f.CdiClient.CdiV1beta1().CDIs().Get(context.TODO(), "cdi", metav1.GetOptions{})
```

Comment:
So I know this code was copied from above, but it is not correct I think, this test should still work if the CDI CR is not named `cdi` there is another function that lists the CDI CRs, and returns the CR if there is only one, regardless of what the name is. This test will be skipped when we deploy CDI with HCO since the CR name will be different, and I don't think that is intended.

Reply:
nice catch, this effectively means we were skipping stuff on runs where cdi.Name != "cdi" :(